tests/serverpod_test_server/test_integration/database_operations/default/string/string_default_persist_test.dart

```diff
@@ -55,5 +55,407 @@ void main() async {
             equals('A specific persist value'));
       },
     );
+
+    // Additional tests for new fields
+
+    test(
+      'when creating a record in the database, then the "stringDefaultPersistSingleQuoteWithOneSingleEscapeQuote" field should match the default value',
+      () async {
+        var object = StringDefaultPersist();
+        var databaseObject = await StringDefaultPersist.db.insertRow(
+          session,
+          object,
+        );
+        expect(
+            databaseObject
+                .stringDefaultPersistSingleQuoteWithOneSingleEscapeQuote,
+            equals('This is a \'default persist value'));
```

Comment:
Suggestion(NIT): Add a comma at the end of the expect for better formatting.

Please apply this to all tests that have bad line breaking in this file.